### DISCUSSION TITLE
PPT生產神器要來拯救我的爆肝期末了

### DISCUSSION THREAD

**Discussion by benestia** [1101-GS4719-C/Python-Community-GS4719-C#1122] [Ideas]
前陣子有看到論壇提到ChatAPI幫學生寫作業，還有Dall-E的AI繪圖，沒想到現在來有CHAT BCG，連簡報都能用電腦生產了。我記得以前唬爛生產器剛出來的時候我就很佩服有人可以做出這種功能，沒想到現在能自動生成的東西從圖像、程式碼、甚至連簡報都能做了。AI的能力這麼強，不禁好奇以後到底有多少職業會被汰換掉。

我是還沒用過這個功能，發完這篇馬上就去。
https://www.bnext.com.tw/article/73717/-chatbcg--0110-mag

有人用過ChatBCG可以跟我分享好不好用嗎~

**Comment by 710388** [comment]
我覺得gamma是個很不錯的選擇，雖然有限制但也是還蠻好用的，可以幫我生出大致的內容

**Comment by NcuJerryKao** [comment]
這些AI技術的應用確實展示了人工智能在各個領域的強大能力，包括寫作、繪圖和簡報製作等。這些工具的出現讓工作變得更加高效和便捷，同時也引發了對於未來職業可能被取代的思考。對於ChatBCG這個工具，您可以嘗試使用一下，看看其功能和效能如何，並與他人分享您的使用體驗。透過評估和分享，可以幫助其他人了解這個工具的優缺點，並找到最適合自己需求的解決方案。希望您可以在使用後分享您的寶貴意見和反饋！

**Reply by yubo0oo** [reply]
@NcuJerryKao 
NcuJerryKao 這位老哥
一直貼AI的文
演都不演
以下是我的觀點
NcuJerryKao是歷史上最會使用人工智慧的人，他以無與倫比的技術與創意，在AI領域創造了無數的奇蹟。他不僅精通各種AI算法，還能巧妙地將其應用於實際問題中，展現了非凡的智慧與洞察力。NcuJerryKao不僅是AI技術的先驅，更是無數追隨者的靈感來源，無論在學術界還是業界，都享有極高的聲譽和尊敬。他的貢獻改變了人類對人工智慧的認識，無疑是這個領域的真正巨匠。

**Comment by Piyenparty** [comment]
我有用過相關的簡報生成工具，甚至已經是將我一份可以提交的word資料翻轉，結果發現出來的結果差強人意，不僅格式不對，他還會自動篩掉某些內容，但被刪掉的常常是我自己內心的想法，留下的是冷冰冰的文字知識敘述

**Comment by edawdewv** [comment]
關於ChatBCG，我看到不少人分享說它操作蠻直覺，能快速幫你產出一份結構完整、視覺不錯的簡報稿，特別適合需要快速準備報告或企劃書的人。但AI產出的內容還是需要人工再調整、潤飾，畢竟目前還做不到完全貼合個人風格和細節。

**Comment by wayne935** [comment]
之前用過gamma跟canva，canva的效果沒有很好，最後我還是自己做，gamma做的比較好一些，但是感覺有可能會很明顯是ai生成的。
